### PR TITLE
[hot fix] enable 2d grouped fwd convolution support on mi300

### PR DESCRIPTION
**Pull request by iq136boy**
Bug fix: enable the 2d grouped fwd solver on mi300 platform

### CODE REVIEW

**Comment by iq136boy** [comment]
@junliume We found that the 2d grouped solver is not enable properly in MI300 platform. This PR is a quick fix to enable the solver on mi300 machines.

**Review by junliume — APPROVED** [review]

File: src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp [hunk 1]
```diff
@@ -454,8 +454,7 @@ bool ConvHipImplicitGemmGroupFwdXdlops::IsApplicable(
     // needed because layout transpose kernel does not support non-packed tensors
     if(problem.IsLayoutDefault() && problem.HasNonPackedTensors())
         return false;
-    const std::string& arch = ctx.GetStream().GetDeviceName();
-    if(!(arch == "gfx908" || arch == "gfx90a"))
+    if(!ck_utility::is_ck_whitelist(ctx.GetStream().GetDeviceName()))
```
Comment (junliume) [inline src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp:458]: @iq136boy we have some issues with this line:
```
[2024-02-19T15:17:40.948Z] /home/jenkins/workspace/MLLIBS_MIOpen_dfeng_gfwd_fix/src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp:457:9: error: use of undeclared identifier 'ck_utility'

[2024-02-19T15:17:40.948Z]     if(!ck_utility::is_ck_whitelist(ctx.GetStream().GetDeviceName()))

[2024-02-19T15:17:40.948Z]         ^
```

Comment (iq136boy) [reply src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp:458]: @junliume let me check.

Comment (junliume) [reply src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp:458]: No problem, PR is updated with missing include file.

**Review by CAHEK7 — APPROVED** [review]

**Pull request merged**